### ISSUE TITLE
Favorite Buildings mod crushes game 

### ISSUE THREAD

**Issue by padlyuck** [O-n-y/oxy.mods#115]
Mod Version: 1.2.7.23 and 1.2.8.27
Game Version: U45-545357-V
OS: Linux
Enabled mods:
 - Mod Manager by Ony
 - Favorite Buildings

The issue causes when attempting to use any shortcut specified in the mod settings.
Maybe important: system was clean before steam and game installation. No other games or saves. 
```
MissingFieldException: Field 'PlanScreen.ActiveCategoryBuildingToggles' not found.

Ony.OxygenNotIncluded.FavoriteBuildings.FavoriteBuildingsComponent.Activate (BuildingDef def) (at <0f6a7d0f5d2c4ceaac5facf54f239e74>:0)
Ony.OxygenNotIncluded.FavoriteBuildings.FavoriteBuildingsComponent.WasActivated (System.String iD) (at <0f6a7d0f5d2c4ceaac5facf54f239e74>:0)
Ony.OxygenNotIncluded.FavoriteBuildings.FavoriteBuildingsComponent.SelectNextBuilding (Ony.OxygenNotIncluded.FavoriteBuildings.HotKeyCategory cat) (at <0f6a7d0f5d2c4ceaac5facf54f239e74>:0)
Ony.OxygenNotIncluded.FavoriteBuildings.FavoriteBuildingsComponent.Update () (at <0f6a7d0f5d2c4ceaac5facf54f239e74>:0)

Build: U45-545357-V
```

**Comment by O-n-y** [comment]
this one need to be updated to last game patch

**Comment by O-n-y** [comment]
the fix will be in upcoming version,  Favorite Buildings  [BETA]  1.2.9.30 is on testing

**Comment by O-n-y** [comment]
fixed on last release  1.2.9.30
